tools/get_release_contributors.py

```diff
@@ -237,21 +248,25 @@ def find_contributors(git_log_selector, from_, to):
         if contributor:
             contributors.add(contributor)
         else:
-            contributors.add(realname)
+            # TODO: check this is the right logic...
```

Comment:
Tagging @hoffie here. If you find this in future, it would be good to verify if the logic here is ok.